### PR TITLE
kdig: add page

### PR DESCRIPTION
**Pull request by dwdh**
- [x] The page (if new), does not already exist in the repository.
- [x] The page is in the correct platform directory (`common/`, `linux/`, etc.)
- [x] The page has 8 or fewer examples.
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- [x] The page follows the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The page description includes a link to documentation or a homepage (if applicable).

**Version of the command being documented (if known):3.0.10**

### CODE REVIEW

File: pages/common/kdig.md [hunk 1]
```diff
@@ -0,0 +1,24 @@
+# kdig
+
+> Advanced DNS lookup utility.
+> More information: <https://manned.org/kdig>.
```
Comment (navarroaxel) [inline pages/common/kdig.md:4]: I think the author's website is better in this case
```suggestion
> More information: <https://www.knot-dns.cz/docs/latest/html/man_kdig.html>.
```

File: pages/common/kdig.md [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+
+- Specify an DNS server to query(e.g. google DNS):
+
+`kdig {{example.com}} @{{8.8.8.8}}`
+
+- Query a specific DNS record type associated with a given domain name:
+
+`kdig {{example.com}} {{A|AAAA|NS|SOA|DNSKEY|ANY}}`
+
+- Get A record for example.com use TLS(DoT):
```
Comment (navarroaxel) [inline pages/common/kdig.md:18]:
```suggestion
- Lookup the IP(s) associated with a hostname (A records) using DNS over TLS (DoT):
```

File: pages/common/kdig.md [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+
+`kdig -d @{{8.8.8.8}} +tls-ca +tls-host={{dns.google}} example.com`
+
+- Get A record for example.com use HTTPS(DoH):
```
Comment (navarroaxel) [inline pages/common/kdig.md:22]:
```suggestion
- Lookup the IP(s) associated with a hostname (A records) using DNS over HTTPS (DoH):
```

File: pages/common/kdig.md [hunk 4]
```diff
@@ -0,0 +1,24 @@
...
+
+`kdig {{example.com}} @{{8.8.8.8}}`
+
+- Query a specific DNS record type associated with a given domain name:
+
+`kdig {{example.com}} {{A|AAAA|NS|SOA|DNSKEY|ANY}}`
+
+- Get A record for example.com use TLS(DoT):
+
+`kdig -d @{{8.8.8.8}} +tls-ca +tls-host={{dns.google}} example.com`
```
Comment (navarroaxel) [inline pages/common/kdig.md:20]:
```suggestion
`kdig -d @{{8.8.8.8}} +tls-ca +tls-host={{dns.google}} {{example.com}}`
```

File: pages/common/kdig.md (hunk 1 above)
Comment (dwdh) [reply pages/common/kdig.md:4]: agree

File: pages/common/kdig.md [hunk 5]
```diff
@@ -0,0 +1,24 @@
+# kdig
+
+> Advanced DNS lookup utility.
+> More information: <https://manned.org/kdig>.
+
+- Get A records for example.com:
```
[the remark this answers is not in the document]
Comment (dwdh) [reply pages/common/kdig.md:6]: ACK

File: pages/common/kdig.md [hunk 6]
```diff
@@ -0,0 +1,24 @@
...
+
+`kdig {{example.com}} {{A|AAAA|NS|SOA|DNSKEY|ANY}}`
+
+- Get A record for example.com use TLS(DoT):
+
+`kdig -d @{{8.8.8.8}} +tls-ca +tls-host={{dns.google}} example.com`
+
+- Get A record for example.com use HTTPS(DoH):
+
+`kdig -d @{{8.8.8.8}} +https +tls-hostname={{dns.google}} example.com`
```
[the remark this answers is not in the document]
Comment (dwdh) [reply pages/common/kdig.md:24]: Of course.
Thanks, I'm new to github and this is my second PR. Made some mistakes, sorry.

[the remark this answers is not in the document]
Comment (navarroaxel) [reply pages/common/kdig.md:24]: np @dwdh 
Remember that you can use this GitHub feature <https://docs.github.com/en/github/collaborating-with-pull-requests/reviewing-changes-in-pull-requests/incorporating-feedback-in-your-pull-request>.
It's the easier way to apply suggestions in `tldr-pages`.

[the remark this answers is not in the document]
Comment (dwdh) [reply pages/common/kdig.md:24]: @svenstaro Thank you.

[the remark this answers is not in the document]
Comment (dwdh) [reply pages/common/kdig.md:24]: @navarroaxel Thank you.

**Comment by CleanMachine1** [comment]
@dwdh Please apply these commits for further review

**Comment by dwdh** [comment]
> @dwdh Please apply these commits for further review

Oh. I'm done.

**Comment by dwdh** [comment]
@CleanMachine1 Oh. I'm done.

**Review by navarroaxel — APPROVED** [review]

**Review by EmilySeville7cfg — APPROVED** [review]

**Comment by EmilySeville7cfg** [comment]
Merged, thanks! ❤️

**Pull request merged**